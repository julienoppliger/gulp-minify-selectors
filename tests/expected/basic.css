.a { display: flex; flex-direction: column; font-family: "Roboto", sans-serif; }
.b { height: 200px; }
.c { font-size: 1.5rem; font-weight: bold; }
.d { font-size: 1.2rem; }