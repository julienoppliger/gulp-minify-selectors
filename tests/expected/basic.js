let card = document.querySelector("a");
