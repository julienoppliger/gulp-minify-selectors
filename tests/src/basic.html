<div class="-s-Card">
	<div class="-s-Card__Image">
	  <img src="assets/images/illustration.jpg">
	</div>
	<div class="-s-Card__Headline">
	  <span>Hello world !</span>
	</div>
	<p class="-s-Card__Text">
	  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
	</p>
</div>
<div class="-s-Card">
	<div class="-s-Card__Image">
	  <img src="assets/images/illustration.jpg">
	</div>
	<div class="-s-Card__Headline">
	  <span>Hello world !</span>
	</div>
	<p class="-s-Card__Text">
	  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
	</p>
</div>