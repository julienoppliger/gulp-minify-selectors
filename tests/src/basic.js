let card = document.querySelector("-s-Card");
