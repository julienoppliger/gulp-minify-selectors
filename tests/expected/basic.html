<div class="a">
	<div class="b">
	  <img src="assets/images/illustration.jpg">
	</div>
	<div class="c">
	  <span>Hello world !</span>
	</div>
	<p class="d">
	  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
	</p>
</div>
<div class="a">
	<div class="b">
	  <img src="assets/images/illustration.jpg">
	</div>
	<div class="c">
	  <span>Hello world !</span>
	</div>
	<p class="d">
	  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
	</p>
</div>